go/vt/sidecardb/schema/vdiff/vdiff.sql

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+CREATE TABLE IF NOT EXISTS _vt.vdiff
+(
+    `id`                 bigint(20)   NOT NULL AUTO_INCREMENT,
```

Comment:
@rohit-nayak-ps For a separate follow up PR, but I think we should drop lengths for `bigint` and other `int` types, since that is deprecated and on MySQL 8.0 doesn't do anything.

Reply:
:+1: and also worth mentioning that `schemadiff` ignores the length values (unless it's `1`), so whether they exist or not, does not make any change to prod.